gammapy/cube/make.py

```diff
@@ -93,10 +98,11 @@ def run(self, observations, selection=None):
             maps_obs = obs_maker.run(selection)
 
             for name in selection:
-                data = maps_obs[name].quantity.to_value(maps[name].unit)
                 if name == "exposure":
+                    data = maps_obs[name].quantity.to_value(maps[name].unit)
                     maps[name].fill_by_coord(obs_maker.coords_etrue, data)
-                else:
+                elif name == "counts" or name == "background":
+                    data = maps_obs[name].quantity.to_value(maps[name].unit)
```

Comment:
Why is this needed? I think it's already handled correctly: https://github.com/gammapy/gammapy/blob/master/gammapy/maps/wcsnd.py#L218